plugins/base/src/main/kotlin/renderers/gfm/GfmRenderer.kt

```diff
@@ -0,0 +1,185 @@
+package org.jetbrains.dokka.base.renderers.gfm
+
+import org.jetbrains.dokka.base.renderers.DefaultRenderer
+import org.jetbrains.dokka.base.resolvers.DefaultLocationProvider
+import org.jetbrains.dokka.base.resolvers.LocationProviderFactory
+import org.jetbrains.dokka.pages.*
+import org.jetbrains.dokka.plugability.DokkaContext
+import java.lang.StringBuilder
+
+
+open class GfmRenderer(context: DokkaContext) : DefaultRenderer<StringBuilder>(context) {
+
+    override fun StringBuilder.buildHeader(level: Int, content: StringBuilder.() -> Unit) {
+        buildParagraph()
+        append("${"#".repeat(level)} ")
+        content()
+        buildNewLine()
+    }
+
+    override fun StringBuilder.buildLink(address: String, content: StringBuilder.() -> Unit) {
+        append("[")
+        content()
+        append("]($address)")
+    }
+
+    override fun StringBuilder.buildList(node: ContentList, pageContext: ContentPage) {
+        buildListLevel(node, pageContext)
+        buildParagraph()
+    }
+
+    private val indent = " ".repeat(4)
+
+    private fun StringBuilder.buildListItem(items: List<ContentNode>, pageContext: ContentPage, bullet: String = "*") {
+        items.forEach {
+            if(it is ContentList) {
+                val builder = StringBuilder()
+                builder.append(indent)
+                builder.buildListLevel(it, pageContext)
+                append(builder.toString().replace(Regex("  \n(?!$)"), "  \n$indent"))
+            } else {
+                append("$bullet ")
+                it.build(this, pageContext)
+                buildNewLine()
+            }
+        }
+    }
+
+    private fun StringBuilder.buildListLevel(node: ContentList, pageContext: ContentPage) {
+        if(node.ordered) {
+            buildListItem(node.children, pageContext, "${node.start}.")
+        } else {
+            buildListItem(node.children, pageContext, "*")
+        }
+    }
+
+    override fun StringBuilder.buildNewLine() {
+        this.append("  \n")
```

Comment:
```suggestion
       append("  \n")
```
no need for an explicit `this` I think

Reply:
Ok